pkg/api/customization/catalog/catalog.go

```diff
@@ -62,6 +62,10 @@ func (a ActionHandler) RefreshActionHandler(actionName string, action *types.Act
 			return err
 		}
 	}
+	data := map[string]interface{}{
+		"catalogs": catalogs,
```

Comment:
Why are we returning this? Just curious about why this is being done for global catalogs too, since global catalog's templates don't seem to have any problems

Reply:
yep, this is not fix for the templates, just a fix for bad refresh response message.
![image](https://user-images.githubusercontent.com/4569037/49262925-7d025c80-f483-11e8-878b-adcddd02f89a.png)

Reply:
yep, this is not fix for the templates, just a fix for bad refresh response message.
![image](https://user-images.githubusercontent.com/4569037/49262925-7d025c80-f483-11e8-878b-adcddd02f89a.png)